### DISCUSSION TITLE
install_routes_xfrmi explanation

### DISCUSSION THREAD

**Discussion by nirdothan** [strongswan/strongswan#1937] [Q&A] [answered]
Hi, 
I'm trying to utilize the new capability with `charon.plugins.kernel-netlink.install_routes_xfrmi`. It doesn't seem to work for me or perhaps I'm not sure what to expect. Is it documented anywhere? 
I have xrfm interfaces with matching `if_id_out` ids, looking at the routing table I don't see any routes installed

**Comment by tobiasbrunner** [comment]
What strongSwan version are you using? This feature is only available since 5.9.10. Also, read the log.

**Reply by nirdothan** [reply, marked as answer]
oh, I'm in  5.9.5
